pkg/api/chunk_stream.go

```diff
@@ -56,7 +56,7 @@ func (s *Service) chunkUploadStreamHandler(w http.ResponseWriter, r *http.Reques
 	}
 
 	// if tag not specified use direct upload
-	putter, err := s.newStamperPutter(r.Context(), putterOptions{
+	putter, err := s.newStamperPutter(context.Background(), putterOptions{
```

Comment:
why is context.Background() better here?

Reply:
probably because the go routine below handles the websocket connection but since this function returns, the context is canceled.

Reply:
yep